sql/item_mods.sql

```diff
@@ -31796,16 +31844,67 @@ INSERT INTO `item_mods` VALUES (16509,500,3);  -- ITEM_ADDEFFECT_DMG: 3
+INSERT INTO `item_mods` VALUES (16510,953,30); -- ITEM_ADDEFFECT_DURATION: 30
+
+-- Colichemarde
+INSERT INTO `item_mods` VALUES (16515,431,1);  -- ITEM_ADDEFFECT_TYPE: DAMAGE
+INSERT INTO `item_mods` VALUES (16515,499,8);  -- ITEM_SUBEFFECT: SUBEFFECT_DARKNESS_DAMAGE
+INSERT INTO `item_mods` VALUES (16515,500,15); -- ITEM_ADDEFFECT_DMG: 15
+INSERT INTO `item_mods` VALUES (16515,501,10); -- ITEM_ADDEFFECT_CHANCE: 10
+INSERT INTO `item_mods` VALUES (16515,950,8);  -- ITEM_ADDEFFECT_ELEMENT: ELEMENT_DARK
+
 -- Verdun
 INSERT INTO `item_mods` VALUES (16520,5,18);  -- MP: 18
 INSERT INTO `item_mods` VALUES (16520,110,7); -- PARRY: 7
 
 -- Rapier
 INSERT INTO `item_mods` VALUES (16521,110,5); -- PARRY: 5
 
+-- Flame Degen
+INSERT INTO `item_mods` VALUES (16522,431,1);  -- ITEM_ADDEFFECT_TYPE: 1
+INSERT INTO `item_mods` VALUES (16522,499,1);  -- ITEM_SUBEFFECT: 1
+INSERT INTO `item_mods` VALUES (16522,500,10); -- ITEM_ADDEFFECT_DMG: 10
+INSERT INTO `item_mods` VALUES (16522,501,5);  -- ITEM_ADDEFFECT_CHANCE: 5
+INSERT INTO `item_mods` VALUES (16522,950,1);  -- ITEM_ADDEFFECT_ELEMENT: 1
+
+-- Holy Degen
+INSERT INTO `item_mods` VALUES (16523,431,1);  -- ITEM_ADDEFFECT_TYPE: DAMAGE
+INSERT INTO `item_mods` VALUES (16523,499,7);  -- ITEM_SUBEFFECT: SUBEFFECT_LIGHT_DAMAGE
+INSERT INTO `item_mods` VALUES (16523,500,15); -- ITEM_ADDEFFECT_DMG: 15
+INSERT INTO `item_mods` VALUES (16523,501,10); -- ITEM_ADDEFFECT_CHANCE: 10
+INSERT INTO `item_mods` VALUES (16523,950,7);  -- ITEM_ADDEFFECT_ELEMENT: ELEMENT_LIGHT
+
+-- Hornet Fleuret
+INSERT INTO `item_mods` VALUES (16525,431,2);  -- ITEM_ADDEFFECT_TYPE: DEBUFF
+INSERT INTO `item_mods` VALUES (16525,499,10); -- ITEM_SUBEFFECT: SUBEFFECT_POISON
+INSERT INTO `item_mods` VALUES (16525,501,10); -- ITEM_ADDEFFECT_CHANCE: 10
+INSERT INTO `item_mods` VALUES (16525,951,3);  -- ITEM_ADDEFFECT_STATUS: EFFECT_POISON
+INSERT INTO `item_mods` VALUES (16525,952,17); -- ITEM_ADDEFFECT_POWER: 17
+INSERT INTO `item_mods` VALUES (16525,953,30); -- ITEM_ADDEFFECT_DURATION: 30
+
 -- Epee
 INSERT INTO `item_mods` VALUES (16527,25,2); -- ACC: 2
 
+-- Bloody Rapier
+INSERT INTO `item_mods` VALUES (16528,431,5);  -- ITEM_ADDEFFECT_TYPE: HP_DRAIN
+INSERT INTO `item_mods` VALUES (16528,499,21); -- ITEM_SUBEFFECT: SUBEFFECT_HP_DRAIN
+INSERT INTO `item_mods` VALUES (16528,500,10); -- ITEM_ADDEFFECT_DMG: 10
+INSERT INTO `item_mods` VALUES (16528,501,5);  -- ITEM_ADDEFFECT_CHANCE: 5
+INSERT INTO `item_mods` VALUES (16528,950,8);  -- ITEM_ADDEFFECT_ELEMENT: ELEMENT_DARK
+
+-- Ancient Sword
+INSERT INTO `item_mods` VALUES (16533,431,2);  -- ITEM_ADDEFFECT_TYPE: DEBUFF
+INSERT INTO `item_mods` VALUES (16533,499,17); -- ITEM_SUBEFFECT: SUBEFFECT_CURSE
+INSERT INTO `item_mods` VALUES (16533,501,10); -- ITEM_ADDEFFECT_CHANCE: 10
+INSERT INTO `item_mods` VALUES (16533,951,9);  -- ITEM_ADDEFFECT_STATUS: EFFECT_CURSE
+INSERT INTO `item_mods` VALUES (16533,953,30); -- ITEM_ADDEFFECT_DURATION: 30
+
```

Comment:
Need a power value for your curse status.

Reply:
Noted. Power added. Best guess is 10%, but will need testing since there is no comments on the power of the curse in any source.